syn/ibex_core_lr_synth_conf.tcl

```diff
@@ -0,0 +1,51 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# List of inputs and outputs.  Number is timing constraint expressed as a % of
```

Comment:
Can you comment (and add that information to the file) on where the values in this file are coming from? Educated guess as a starting point, or is there more behind it?